### PR TITLE
Added annotation about importance of stringtable.xml

### PR DESCRIPTION
**Pull request by Smieszkokoleszko**
Title says everything.

### CODE REVIEW

File: docs/guides/missionmaking.md [hunk 1]
```diff
@@ -112,6 +112,14 @@ _Jeżeli zamierzasz bawić się skryptami to bardzo polecamy `Visual Studio Code
 
 W naszych misjach pojawiają się różne treści. Takie rzeczy można przygotowywać w pliku stringtable.xml. Misja pokazowa ma tam zdefiniowaną nazwę nazwę misji oraz treść intelu. Najprościej przyjąć, że to tablica stałych tekstowych.
 
+**UWAGA**
+Nie pomijaj tego punktu. Jeśli nie masz potrzeby niczego zmieniać przynajmniej edytuj nazwę misji. W miejsce `ArmaForces - Preset` wpisz swoją unikalną nazwę misji. Unikniesz dzięki temu problemów z dublowaniem się nazw na serwerze i tym samym brakiem Twojej misji na liście. Najlepiej wyeduj wszystkie pola, oprócz korzyści czysto porządkowych gracze zobaczą większe napracowanko.
```
Comment (3Mydlo3) [inline docs/guides/missionmaking.md:116]: > Najlepiej wyeduj wszystkie pola
I guess it should be `edytuj` or `wyedytuj` although the second one seems a bit odd.

Comment (3Mydlo3) [inline docs/guides/missionmaking.md:116]: > Najlepiej wyeduj wszystkie pola, oprócz korzyści czysto porządkowych gracze zobaczą większe napracowanko. 
I think these should be 2 separate sentences. This doesn't sound natural.

File: docs/guides/missionmaking.md [hunk 2]
```diff
@@ -112,6 +112,14 @@ _Jeżeli zamierzasz bawić się skryptami to bardzo polecamy `Visual Studio Code
...
 W naszych misjach pojawiają się różne treści. Takie rzeczy można przygotowywać w pliku stringtable.xml. Misja pokazowa ma tam zdefiniowaną nazwę nazwę misji oraz treść intelu. Najprościej przyjąć, że to tablica stałych tekstowych.
 
+**UWAGA**
+Nie pomijaj tego punktu. Jeśli nie masz potrzeby niczego zmieniać przynajmniej edytuj nazwę misji. W miejsce `ArmaForces - Preset` wpisz swoją unikalną nazwę misji. Unikniesz dzięki temu problemów z dublowaniem się nazw na serwerze i tym samym brakiem Twojej misji na liście. Najlepiej wyeduj wszystkie pola, oprócz korzyści czysto porządkowych gracze zobaczą większe napracowanko. 
+```xml
+<Key ID="STR_AFP_Scripts_DisplayName">
+	<Polish>ArmaForces - Preset</Polish>
+</Key>
+```
+
```
Comment (3Mydlo3) [inline docs/guides/missionmaking.md:122]: I think it could be nice to elaborate a bit on how to create new stringtable keys.

Each key should contain `STR_AFP_Scripts` part unchanged, otherwise it won't work correctly (or at all). And mention that language highest on the list is the default one if correct language is not available.

If the configuration would be like below, then someone with German language would see Polish strings (which is nonsense most of the time).
```xml
<Key ID="STR_AFP_Scripts_SomeString">
    <Polish>Jakiś string</Polish>
    <English>Some string</English>
</Key>
```

**Review by 3Mydlo3 — CHANGES_REQUESTED** [review]

**Comment by Smieszkokoleszko** [comment]
New commit notification.

File: docs/guides/missionmaking.md [hunk 3]
```diff
@@ -112,8 +112,31 @@ _Jeżeli zamierzasz bawić się skryptami to bardzo polecamy `Visual Studio Code
...
+A następnie wywołaj go w innych plikach, np. `tasks.hpp` w ten sposób:
+```hpp
+title = "STR_AFP_Scripts_Task_NazwaCelu";
+description = "STR_AFP_Scripts_Task_OpisCelu";
+```
+
+Pierwszy język pojawiający się w kluczu jest uznawany za domyślny, dlatego jeśli robisz misję dla grupy międzynarodowej stawiaj angielski na pierwszym miejscu, wtedy jeśli nie masz tłumaczenia na język gracza tekst wyświetli mu się w języku angielskim.</br>
+Oczywiście robiąc misję dla grupy gdzie wszyscy gracze mówią w jednym języku nie ma potrzeby dodawania tłumaczeń. Szczególnie, że wiele osób ma ustawiony język na angielski i widziałoby te opisy po angielsku. 
+
+Być może trafisz w internecie na jakieś przykłady gdzie zmienne tekstowe poprzedzane są symbolem `$`. Unikaj używania tego i stosuj przykład pokazany wyżej, bo inaczej możesz mieć problemy z prawidłowym tłumaczeniem tekstów.
```
Comment (3Mydlo3) [inline docs/guides/missionmaking.md:136]: Not exactly. I think it should be more like:
```suggestion
Być może trafisz w internecie na jakieś przykłady gdzie zmienne tekstowe w configu poprzedzane są symbolem `$`. Powoduje to, że tekst jest od razu tłumaczony przy ładowaniu do configu. Dlaczego więc w przykładzie jest bez `$`? Ponieważ serwer tworzy zadania i rozsyła je graczom. Jeżeli tekst będzie już przetłumaczony to gracze będą widzieć zadania w języku serwera. Jeżeli zostawimy bez `$` to gracze otrzymają klucz do stringtable i skrypt przetłumaczy tekst lokalnie u gracza.

TLDR w taskach bez `$`.
```

**Review by 3Mydlo3 — CHANGES_REQUESTED** [review]
